docs/source/variables.adoc

```diff
@@ -1,2 +1,2 @@
-:minishift-version: unset
-:openshift-version: unset
+:minishift-version: 1.0.0-rc.1
```

Comment:
-1, variables.adoc should not even be checked in. If it is, then it should be removed. Might have been accidentally added at some stage. 

It is in .gitignore, so I don't know how it ended up there.